scrabble.js

```diff
@@ -1,16 +1,202 @@
 
 const Scrabble = {
+
+
   score(word) {
 
+    let wordScore = 0
+
+    const SCORING_RUBRIK = {
+      'a': 1,
+      'e': 1,
+      'i': 1,
```

Comment:
I like the use of a hash to store this information - good instincts! However, defining it inside the method ends up being a little cumbersome. Putting it at the top of the file might be a little more readable.